packages/block-editor/src/components/block-mobile-toolbar/block-actions-menu.native.js

```diff
@@ -426,12 +422,12 @@ export default compose(
 					);
 				},
 				duplicateBlock() {
-					return duplicateBlocks( clientIds );
+					return duplicateBlocks( [ clientId ] );
 				},
 				onMoveDown: ( ...args ) =>
-					moveBlocksDown( clientIds, rootClientId, ...args ),
+					moveBlocksDown( clientId, rootClientId, ...args ),
```

Comment:
Won't this fail when attempting to verify that blocks can be moved, because `canMoveBlocks()` still expects an array?

https://github.com/WordPress/gutenberg/blob/8f35cbdddd1341f307f7c53c688239a6700b743f/packages/block-editor/src/store/actions.js#L448

Reply:
You're right, It turns out I was testing the same functionality but from another component which was not this one 😓 I've updated the code and the test accordingly in https://github.com/WordPress/gutenberg/pull/46699/commits/82c2b8598dcdba58f59eb5da820c6f5544f49559

Thanks for catching that!